policies/branch-policy.md

```diff
@@ -0,0 +1,97 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+### The default development branch
+
+- Any type (bug fix, feature, refactoring, ...) of pull requests is allowed.
+- The development of the next minor or major release happens there.
+  Which kind of release it is is defined by OMC at some point after the
+  branch is opened after the previous release was branched.
+  Implicitly it means that any API/ABI breaking changes are not allowed
+  on this branch unless OMC defines the next release will be a major
+  release.
+- Any changes merged to this branch must be ported to the future major
+  and future minor branches if they are applicable. This can be done by
```

Comment:
Should this implicitly say changes need to be backported to the supported release branches also.. (It is currently implied since changes to the future branches are backported).

Reply:
H

Reply:
IMO no, because this applies only to bug fixes. And backporting bug fixes is handled on line 28.